cmd/cql/internal/transfer.go

```diff
@@ -0,0 +1,95 @@
+package internal
+
+import (
+	"encoding/json"
+	"regexp"
+	"strconv"
+
+	"github.com/CovenantSQL/CovenantSQL/client"
+	"github.com/CovenantSQL/CovenantSQL/crypto/hash"
+	"github.com/CovenantSQL/CovenantSQL/proto"
+	"github.com/CovenantSQL/CovenantSQL/types"
+)
+
+var CmdTransfer = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdTransfer should have comment or be unexported